awesome_owl/static/src/playground.scss

```
@@ -0,0 +1,3 @@
+.chapter_tuto {
+    border: solid; padding: 10px; border-radius: 5px; margin-bottom: 5px
+}
```

just for lisibility :)

```suggestion
.chapter_tuto {
    border: solid; 
    padding: 10px; 
    border-radius: 5px; 
    margin-bottom: 5px
}
```